CDP4Common/Poco/NestedElement.cs

```diff
@@ -74,5 +74,82 @@ protected DomainOfExpertise GetDerivedOwner()
         /// is the root of the Nested tree.
         /// </summary>
         public bool IsRootElement { get; internal set; }
+
+        /// <summary>
+        /// The <see cref="ElementBase"/> representing a <see cref="NestedElement"/>.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="ElementBase"/>.
+        /// </returns>
+        public ElementBase GetElementBase()
+        {
+            return this.IsRootElement
+                ? (ElementBase)this.RootElement
+                : this.ElementUsage.Last();
```

Comment:
not sure i understand the `Last()` method here? i propose we have a short conversation about that

Reply:
Take a look at the elementUsage property here: [SDK Reference](https://sdk-reference.cdp4.org/#NestedElement)
elementUsage contains an ordered list of hierarchical elementUsages up to the TopElement. So the last ElementUsage will be the correct ElementUsage that the NestedElement represents.
Last() method was already used in the (handcoded POCO) NestedElement.GetDerivedOwner() method.

Reply:
ok, i sort of remember, thanks for the link to the reference.